src/components/binding/binding-loader.js

```diff
@@ -180,12 +180,12 @@ genotet.BindingLoader.prototype.loadExons_ = function(chr) {
 
 /**
  * Queries the locus of a given gene.
- * @param {string} gene Gene name to be searched for.
+ * @param {string} fileName File name to be searched for.
```

Comment:
Incorrect change. findLocus indeed uses "gene" rather than "fileName" because it reads from exons not wiggle files.